ethcore/types/src/transaction/transaction.rs

```diff
@@ -237,7 +234,8 @@ impl Transaction {
 		}
 	}
 
-	/// Add EIP-86 compatible empty signature.
+	/// Legacy EIP-86 compatible empty signature.
+	/// Used only for tests.
```

Comment:
I'm not happy with keeping this method public, but `cfg(test)` doesn't work across crates :(

Reply:
`#[cfg(feature = "test-helpers")]` ?

Reply:
`#[cfg(any(test, feature = "test-helpers"))]` is what we've used elsewhere (not pretty for sure).